src/ops/nn.jl

```diff
@@ -204,10 +204,23 @@ end
 @not_implemented function all_candidate_sampler()
 end
 
-@not_implemented function atrous_conv2d()
+@not_implemented function atrous_conv2d(value, filters, rate, padding; name="")
+    desc = NodeDescription("AtrousConv2D", get_name(name))
```

Comment:
In the new style, this should be something like

```julia
local desc
with_op_name(name) do
    desc = NodeDescription("AtrousConv2D")
end
```

instead of using the two-argument `NodeDescription` constructor.